doc/source/examples/ale_regression_california.ipynb

```diff
@@ -0,0 +1,875 @@
+{
```

Comment:
<p>In the last sentence 'a smaller median income in the are bring the predicted house value down with respect to the average prediction.' shouldn't it be ' ... average feature effect'?</p><p> </p><p>in the are bring -&gt; in the area brings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>